pom.xml

```diff
@@ -150,6 +150,17 @@
             <version>1.4.10</version>
             <scope>compile</scope>
         </dependency>
+        <!-- Core GTFS Library -->
+        <dependency>
+            <groupId>org.onebusaway</groupId>
+            <artifactId>onebusaway-gtfs</artifactId>
+            <version>1.3.4</version>
```

Comment:
The most recent version is `1.3.94`, but to get it you'll need to add the Camsys maven repo as a repository as the location of the JARs moved:

```
    <repository>
      <id>releases-camsys-public-repo</id>
      <url>https://repo.camsys-apps.com/releases/</url>
      <releases>
        <enabled>true</enabled>
      </releases>
      <snapshots>
        <enabled>false</enabled>
      </snapshots>
    </repository>
```

You can see this at https://github.com/OneBusAway/onebusaway-gtfs-modules/blob/master/pom.xml#L35.